templates/terraform/examples/subnetwork_logging_config_beta.tf.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_compute_subnetwork" "subnet-with-logging" {
```

Comment:
I think we need a [`google-beta` block](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/examples/autoscaler_single_instance.tf.erb#L75-L78) as well?

Reply:
I don't think we do? Tests pass currently. I'm happy to add one though

Reply:
Other beta provider tests (region_autoscaler, vpn_tunnel) don't have a `google-beta` block

Reply:
I think I see one on them? eg https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/examples/vpn_tunnel_beta.tf.erb#L72

Reply:
It's entirely possible that `0.12` somehow made the block no longer necessary. Weird!

Reply:
Ah, it was at the bottom! Added it

Reply:
Just confirmed this as well, leaving them undefined newly works as of using the `0.12` SDK. Since OiCS is still on `0.11`, we should define them explicitly for now.